server/services/v1/api.go

```diff
@@ -42,6 +43,8 @@ const (
 
 	documentPath        = collectionPath + "/documents"
 	documentPathPattern = documentPath + "/*"
+
+	infoPath = "/info"
```

Comment:
but why not call it version?

Reply:
what's the path look like?

Reply:
`/api/v1/info` 

we had a long discussion on this one here https://github.com/tigrisdata/tigris-api/pull/46#discussion_r862020814

Reply:
I think it's good idea to have more generic `info` where we can put any other server related metadata.

Reply:
Makes sense, let's go with info.